pubspec.yaml

```diff
@@ -17,5 +17,5 @@ dev_dependencies:
   build_test: ^0.10.0
   build_web_compilers: '>=0.3.1 <3.0.0'
   dart_style: ^1.0.9
-  pedantic: '^1.4.0'
+  pedantic: ^1.4.0
```

Comment:
Were the lint fixes part of a package pedantic update or just some that were already violations in the existing version?

Reply:
The lint fixes were already in violation. https://travis-ci.org/dart-lang/observable/builds